pkg/registry/apis/dashboard/large.go

```diff
@@ -14,12 +14,12 @@ import (
 	"github.com/grafana/grafana/pkg/storage/unified/apistore"
 )
 
-func NewDashboardLargeObjectSupport(scheme *runtime.Scheme) *apistore.BasicLargeObjectSupport {
+func NewDashboardLargeObjectSupport(scheme *runtime.Scheme, ts int) *apistore.BasicLargeObjectSupport {
 	return &apistore.BasicLargeObjectSupport{
 		TheGroupResource: dashboardV0.DashboardResourceInfo.GroupResource(),
 
-		// byte size, while testing lets do almost everything (10bytes)
-		ThresholdSize: 10,
+		// Byte size after which an object is considered large.
+		ThresholdSize: ts,
```

Comment:
Would it make sense to have the unit in the name of the variable/config? `ThresholdSizeBytes`

Reply:
Good idea: [c08e87d](https://github.com/grafana/grafana/pull/101774/commits/c08e87dd91c54bcefae418f510d98e187f4e9f9a)